geotrek/zoning/tests/test_parsers.py

```diff
@@ -59,3 +61,100 @@ def test_multipolygon(self):
         geom = MultiPolygon(Polygon(((0, 0), (0, 1), (1, 1), (1, 0), (0, 0))))
         self.assertEqual(self.parser.filter_geom("geom", geom), geom)
         self.assertFalse(self.parser.warnings)
+
+
+class TestDistrictOpenStreetMapParser(OpenStreetMapDistrictParser):
+    provider = "OpenStreetMap"
+    tags = [[{"boundary": "administrative"}, {"admin_level": "6"}]]
+
+
+class OpenStreetMapDistrictParser(TestCase):
+    @mock.patch("geotrek.common.parsers.requests.get")
+    def import_district(self, nominatim_file, mocked):
+        def mocked_json_overpass():
+            filename = os.path.join(
+                os.path.dirname(__file__), "data", "district_OSM.json"
+            )
+            with open(filename) as f:
+                return json.load(f)
+
+        def mocked_json_nominatim():
+            filename = os.path.join(os.path.dirname(__file__), "data", nominatim_file)
+            with open(filename) as f:
+                return json.load(f)
+
+        response1 = mock.Mock()
+        response1.json = mocked_json_overpass
+        response1.status_code = 200
+
+        response2 = mock.Mock()
+        response2.json = mocked_json_nominatim
+        response2.status_code = 200
+
+        mocked.side_effect = [response1, response2]
+
+        call_command(
+            "import",
+            "geotrek.zoning.tests.test_parsers.TestDistrictOpenStreetMapParser",
+        )
+
+        self.district = District.objects.order_by("pk").all()
```

Comment:
Considering that `self.district` is a queryset, I think it should be `self.districts` so it's clear that it's not just one object, even though currently the queryset only contains one (the tests could evolve).

```suggestion
        self.districts = District.objects.order_by("pk").all()
```